ceph.spec.in

```diff
@@ -46,7 +46,6 @@
 %endif
 %endif
 %bcond_without selinux
-%bcond_without cephfs_java
```

Comment:
It would be better to comment this line off instead of deleting it and mention the reason for disabling this option.

Reply:
Mostly noone is depending on it - I'll vote for removal.

Reply:
are we planning to remove it from cmake and deb builds too?

Reply:
we should  - no point in carrying stuff we know is not _at all_ being used.